package.json

```diff
@@ -37,22 +37,22 @@
     "onCommand:PowerShell.RestartSession"
   ],
   "dependencies": {
-    "vscode-languageclient": "3.3.0"
+    "vscode": "^1.1.18",
```

Comment:
How come this was moved to being a dependency vs dev dependency?

Reply:
Could this be the cause of that unfriendly error?

Reply:
This is based on the VSCode tutorial on how to write a language server - https://code.visualstudio.com/docs/extensions/example-language-server

There is a section that displays the `dependencies` you would have in your package.json file as:
```
"dependencies": {
    "vscode": "^1.1.5",
    "vscode-languageclient": "^3.3.0"
}
```
RE that error - no, that was pilot error on my part because I forgot that you have to run `npm run postinstall` right after you run `npm install` and before you try to run `npm run compile`.

Reply:
Hmmm I wonder if that was just to make things simple. Even `vscode-languageclient` itself has `vscode` as a dev dependency:
https://www.npmjs.com/package/vscode-languageclient

Reply:
I could remove `vscode` and see if it still compiles.  Want me to do that?

Reply:
It seems that the `vscode` dependency is necessary, in addition to `vscode-languageclient`.  Without the former I get this error at the end of `npm install`:
```
Error: Cannot find module 'C:\Users\Keith\GitHub\rkeithhill\vscode-powershell\node_modules\vscode\bin\install'
```